docsource/functionalRequirements.rst

```diff
@@ -1,2 +1,24 @@
 Functional Requirements
-========================
\ No newline at end of file
+========================
+
+1. Support the indexing of many data types (Genome, Assembly, Narrative etc). It is expected that there will be about 100 types
+
+2. Support the indexing of versions of types. It is expected that each object can have on average about 100 versions.
+
+3. Support the indexing of instance versions. A specific version of a specific object may have mutable data that me require index updates.
```

Comment:
typo me > may

Reply:
I'm not sure what instance versions means?

Reply:
Fixed typo. Changed to "Support the indexing of multiple versions of an object"